--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,7 +42,7 @@
 gix-faster = ["gix-features/zlib-stock", "gix/fast-sha1"]
 
 [dependencies]
-chrono = { version = "0.4.24", default-features = false, features = ["clock", "std", "wasmbind"] }
+chrono = { version = "0.4.24", default-features = false, features = ["clock", "std", "wasmbind"], git = "https://github.com/RumovZ/chrono.git", branch = "android-tzdata-crate" }
 clap = { version = "4.3.0", features = ["derive", "cargo", "unicode"] }
 clap_complete = "4.3.0"
 dirs-next = "2.0.0"
